doc/connectivity/bluetooth/api/mesh/brg_cfg.rst

```diff
@@ -113,13 +113,16 @@ protection to ensure network security. Key considerations to take into account a
 Relay buffer considerations
 ===========================
 
-When a message is relayed between subnets by a Subnet Bridge, it is allocated from the relay buffer.
-To ensure that messages can be retransmitted to all subnetworks,
-the :kconfig:option:`CONFIG_BT_MESH_RELAY_BUF_COUNT` option should be increased accordingly.
-
-However, if the :kconfig:option:`CONFIG_BT_MESH_RELAY` feature is disabled, the messages will be
-allocated from the advertising buffer instead. In this case, increase the
-:kconfig:option:`CONFIG_BT_MESH_ADV_BUF_COUNT` option to allow for sufficient buffer space.
+When a message is relayed between subnets by a Subnet Bridge, it is allocated from the relay buffer
+pool and transmitted using the relay advertising sets. These will be used even if the relay feature
+:kconfig:option:`CONFIG_BT_MESH_RELAY` is disabled. If enabled, the relay buffers and advertising
+sets will use the parameters set by :kconfig:option:`CONFIG_BT_MESH_RELAY_ADV_SETS` and
+:kconfig:option:`CONFIG_BT_MESH_RELAY_BUF_COUNT`. If relaying is disabled, the parameters set by
+:kconfig:option:`CONFIG_BT_MESH_BRG_ADV_SETS` and :kconfig:option:`CONFIG_BT_MESH_BRG_BUF_COUNT`
+will be used.
```

Comment:
May I suggest something else? In my mind, I always assumed that "If relay feature is disabled, SBR will still automatically use CONFIG_BT_MESH_RELAY_ADV_SETS and CONFIG_BT_MESH_RELAY_BUF_COUNT. 

Additional CONFIG_BT_MESH_BRG_ADV_SETS, and CONFIG_BT_MESH_BRG_BUF_COUNT does not really provide a real value. As a product user, you would like to have a similar performance for relay and a bridge, which means ... both of these functions should ideally use a single KConfig options for controlling sets and buffer counts. We can discuss this offline if needed.

Reply:
Hmm I guess we could extract `BT_MESH_RELAY_ADV_SETS` and `BT_MESH_RELAY_BUF_COUNT` and let them be visible if `BT_MESH_RELAY` or `BT_MESH_BRG_CFG_SRV` is enabled. I'm not sure where we would place them though, as this might affect the menuconfig. Any suggestions?

Reply:
Pushed a suggestion for this.